sample-apps/communication/main.js

```diff
@@ -22,16 +23,26 @@ async function init () {
         msgElement.style.display = "none"
       }, 5000)
     }
+    // TODO(orenco): promise syntax
```

Comment:
Add more information for future generations

Reply:
Added